common/fileview2/fvresultset.cpp

```diff
@@ -3309,11 +3309,11 @@ extern FILEVIEW_API void writeFullWorkUnitResults(const char *username, const ch
 
             writer.outputBeginNested(getSeverityTagname(severity, flags), false);
             if (code)
-                writer.outputUInt(code, "Code");
+                writer.outputUInt(code, 0, "Code");
```

Comment:
(Although the function currently only checks if < 7, that doesn't mean another implementation may not do something else with it.)